examples/cpp/pyperf/PyPerfCollapsedPrinter.cc

```diff
@@ -78,41 +77,40 @@ void PyPerfCollapsedPrinter::processSamples(
   for (auto& sample : samples) {
     int frames = 0;
     std::fprintf(output_file, "%s-%d/%d", sample.comm.c_str(), sample.pid, sample.tid);
+
+    for (auto it = sample.pyStackIds.crbegin(); it != sample.pyStackIds.crend(); ++it) {
+      const auto stackId = *it;
+      auto symbIt = symbols.find(stackId);
+      if (symbIt != symbols.end()) {
+        std::fprintf(output_file, ";%s", symbIt->second.c_str());
+        frames++;
+      } else {
+        std::fprintf(output_file, ";%s", kLostSymbol.c_str());
+        lostSymbols++;
+      }
+    }
+
     switch (sample.stackStatus) {
     case STACK_STATUS_TRUNCATED:
-      std::fprintf(output_file, ";%s", kIncompleteStack.c_str());
+      std::fprintf(output_file, ";%s", kTruncatedStack.c_str());
```

Comment:
These 2 should be printed before the stacks themselves

Reply:
Done